packages/api/tests/lambdas/ems-distribution-report.js

```diff
@@ -6,11 +6,47 @@ const path = require('path');
 const test = require('ava');
 const { aws } = require('@cumulus/common');
 const { testUtils: { randomString } } = require('@cumulus/common');
-const { generateAndStoreDistributionReport } = require('../../lambdas/ems-distribution-report');
+const { bucketsPrefixes, generateAndStoreDistributionReport } = require('../../lambdas/ems-distribution-report');
+const models = require('../../models');
+const { fakeGranuleFactory, fakeFileFactory } = require('../../lib/testUtils');
+
+let granuleId;
+
+test.before(async () => {
+  process.env.system_bucket = randomString();
+  process.env.stackName = 'test-stack';
+  process.env.ems_provider = 'testEmsProvider';
+
+  process.env.GranulesTable = randomString();
+  process.env.FilesTable = randomString();
+
+  const fileModel = new models.FileClass();
+  const granuleModel = new models.Granule();
+  await granuleModel.createTable();
+  await fileModel.createTable();
+
+  // add file and granule
+  const bucket = 'my-dist-bucket';
```

Comment:
Could there be some foot stepping here if multiple stacks are running this integration test at once? May be a good idea to either make this a random string or somehow unique to the run.